src/main/kotlin/team/jeonghokim/zakbu/domain/routine/presentation/RoutineController.kt

```diff
@@ -0,0 +1,20 @@
+package team.jeonghokim.zakbu.domain.routine.presentation
+
+import jakarta.validation.Valid
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import team.jeonghokim.zakbu.domain.routine.presentation.dto.request.CreateRoutineRequest
+import team.jeonghokim.zakbu.domain.routine.service.CreateRoutineService
+
+@RestController
+@RequestMapping("/routines")
+class RoutineController(
+    private val createRoutineService: CreateRoutineService
+) {
+    @PostMapping
```

Comment:
요청이 성공했을시 201이 반환되도록 @ResponseStatus를 추가하는것이 좋을 것 같습니다!
```suggestion
    @PostMapping
    @ResponseStatus(HttpStatus.CREATED)
```